.github/workflows/easysplash.yml

```diff
@@ -0,0 +1,15 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: install dependencies
+      run: sudo apt-get install -y libsystemd-dev
+    - name: make
+      run: mkdir build && cd build && cmake .. -DDISPLAY_TYPE_SWRENDER=1 && make
```

Comment:
@texierp thanks for looking into this.

I think you could use something as:

```
cmake -S . -B build -DDISPLAY_TYPE_SWRENDER=1
cmake --build build
```

Reply:
Indeed @otavio ;)

PR updated !